src/fmi4c.c

```diff
@@ -1408,7 +1410,7 @@ bool loadFunctionsFmi1(fmiHandle *fmu)
     getcwd(cwd, sizeof(char)*FILENAME_MAX);
 #endif
 
-    char *dllPath = (char*)malloc(sizeof(char)*FILENAME_MAX);
+    char dllPath[FILENAME_MAX];
```

Comment:
Not sure why I used malloc here 😕 But I guess the result will be equivalent, just more readable?

Reply:
Auto cleanup of memory when function return, else memory leek if free is not called manually